portal/src/views/shared/StationHoverSummary.vue

```diff
@@ -109,7 +109,7 @@ export default Vue.extend({
                 const level = this.stationFeature.properties.thresholds?.levels.find(
                     (level) => level.start < primaryValue && level.value > primaryValue
                 );
-                return level?.label["enUS"];
+                return level?.plainLabel["enUS"];
```

Comment:
Will this fall back on another label if plainLabel isn't defined?

Reply:
Now it does